src/hotspot/cpu/riscv/macroAssembler_riscv.cpp

```diff
@@ -1503,6 +1523,17 @@ static address get_target_of_movptr(address insn_addr) {
   return (address) target_address;
 }
 
+static address get_target_of_movptr2(address insn_addr) {
```

Comment:
Similar here. Maybe we can have a common entry of `get_target_of_movptr` which delegates work to `get_target_of_movptr1` and `get_target_of_movptr2`?

Reply:
Same here or?

Reply:
Yeah, just keep the current shape.